src/contracts/IEventReturnContext.ts

```diff
@@ -0,0 +1,6 @@
+import { MessageEmbed } from "discord.js";
```

Comment:
Since this will be the same once `ICommandReturnContext` is changed, merge these into a `IReturnContext` maybe